extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -86,6 +86,8 @@ const iconStyle = (required) => {
   };
 };
 
+const resourceFieldStyles = { root: { paddingBottom: '6px', width: '75%' } };
```

Comment:
We should try to use multiples of 4.
```suggestion
const resourceFieldStyles = { root: { paddingBottom: '8px', width: '75%' } };
```